boxcli/midcobra/sentry.go

```diff
@@ -0,0 +1 @@
+package midcobra
```

Comment:
Should this file be deleted?

Reply:
oops, yes, it doesn't belong in this PR. I use this file in the next one.